test/react-native-cli/features/cli-tests/automate-symbolication.feature

```diff
@@ -3,33 +3,33 @@ Feature: automate symbolication command
 Scenario: successfully modify project
     When I run the React Native service interactively
     And I input "bugsnag-react-native-cli automate-symbolication" interactively
-    Then I wait for the shell to output a line containing "No repo detected." to stdout
-    And I wait for the shell to output the following to stdout
+    Then I wait for the shell to output a match for the regex "No repo detected." to stdout
```

Comment:
A lot of these steps are using regex special characters (`.` and `?`) so they don't quite match what they're expected to. It'd be good to escape those so it's clearer what's happening in case a future addition causes issues by being too loose

e.g.

```ruby
irb(main):001:0> a = Regexp.new("Are you using Bugsnag on-premise?")
irb(main):002:0> a.match? "Are you using Bugsnag on-premis"
=> true
irb(main):003:0> b = Regexp.new("Are you using Bugsnag on-premise\\?")
irb(main):004:0> b.match? "Are you using Bugsnag on-premis"
=> false
irb(main):005:0> b.match? "Are you using Bugsnag on-premise"
=> false
irb(main):006:0> b.match? "Are you using Bugsnag on-premise?"
=> true
```